src/rulesets/operations.ts

```diff
@@ -45,6 +50,7 @@ export const rules = {
     operations.requirement.must(
       "use the correct case",
       (operation, context, docs, specItem) => {
+        // TODO: did not find a doc link for this
```

Comment:
https://github.com/snyk/sweater-comb/blob/main/docs/standards.md#parameter-names-and-path-components